nuxt/stories/productStage.stories.js

```diff
@@ -165,3 +165,33 @@ export const Cards = {
     },
   },
 };
+
+export const Slice = {
+  args: {
+    stage: {
+      bgColor: 'var(--color-orange);',
+      headline: 'Start now',
+      description:
```

Comment:
There is another link inside conditional-link but this one does nothing. What is it for?